src/gpuarray_array_collectives.c

```diff
@@ -10,7 +10,8 @@
  */
 static inline size_t find_total_elems(const GpuArray* array) {
   unsigned int i;
-  size_t total_elems = 1;
+  size_t total_elems;
+  total_elems = 1;
```

Comment:
I don't remember. Probably not, but I remembered what you said about mvsc
and assigning when declaring. Except if this only happens in 'for' clauses.
Then it's unnecessary :p

On Jul 20, 2016 19:36, "abergeron" <notifications@github.com> wrote:

> In src/gpuarray_array_collectives.c
> <https://github.com/Theano/libgpuarray/pull/223#discussion_r71560329>:
>
> > @@ -10,7 +10,8 @@
> >   */
> >  static inline size_t find_total_elems(const GpuArray* array) {
> >    unsigned int i;
> > -  size_t total_elems = 1;
> > +  size_t total_elems;
> > +  total_elems = 1;
>
> Was this really triggering the warning?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/libgpuarray/pull/223/files/57831c31cb08adc2bc9991f30455f9ce3ed882ed..0c284872b54061a0ee65543c3b63562e6f546a0a#r71560329>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD5VATWKgrgleiRM2TWBdioEQcWR90e0ks5qXk6FgaJpZM4JOG-U>
> .
>